### DISCUSSION TITLE
Failed to run `git --version`. Git may not be properly installed:   '--version' is not recognized as an internal or external command,;operable program or batch file.

### DISCUSSION THREAD

**Discussion by americanslon** [devlooped/GitInfo#276] [Q&A]
GitInfo 3.0.4
VS Enterprise 2022 17.5.1

We are a fully MS shop using Azure/Devops so I never manually installed Git. We are just using whatever installation/plug in that comes with the Studio. Am I misunderstanding something about how this is supposed to work?

Thank you!

**Comment by lucas-0liveira** [comment]
It sounds like you're saying that your organization primarily relies on Microsoft technologies, particularly Azure and DevOps. In this environment, you haven't had the need to manually install Git because you're using the installation or plugin that comes bundled with Visual Studio.

You're wondering if there might be a misconception about how this setup is supposed to function. If I understand correctly, you're seeking confirmation or clarification on whether this is the expected and correct way to handle Git within your specific technological environment.

**Comment by kzu** [comment]
If you can open a regular command prompt and run `git --version`, GitInfo will work.
